File: app/src/main/java/ntutifm/game/google/net/ApiClass/OilStation.java
package ntutifm.game.google.net.ApiClass;

import com.google.gson.annotations.SerializedName;

public class OilStation {
    @SerializedName("Station")
    private String station ="station";
    @SerializedName("Longitude")
    private Double longitude = 123.1;
    @SerializedName("Latitude")
    private Double latitude = 25.2;
    @SerializedName("Address")
    private String address = "address";


    public String getStation() {
        return this.station;
    }
    public String getAddress() {
        return this.address;
    }
    public Double getLatitude() {return this.latitude;}
    public Double getLogitude() {
        return this.longitude;
    }

}

File: app/src/main/java/ntutifm/game/google/apiClass/OilStation.kt
package ntutifm.game.google.apiClass

import com.google.gson.annotations.SerializedName

data class OilStation (
    @SerializedName("Station") val station: String,
    @SerializedName("Longitude") val logitude: Double,
    @SerializedName("Latitude") val latitude: Double,
    @SerializedName("Address") val address: String
)